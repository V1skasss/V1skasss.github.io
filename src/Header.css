.header {
  background-color: #440A0A;
  display: flex;
  justify-content: space-between;
  align-items: center;
  height: 100px;
  padding: 0 20px;
}

.logo-and-title {
  display: flex;
  align-items:center;
}

.logo img {
  width: 160px; 
  height: 150px;
  margin-left: -15px; 
}

.title {
  font-size: 17px; 
  color: white;
  margin-left: -50px;
}


nav ul {
  list-style-type: none;
  display: flex;
}

nav ul li {
  margin-right: 40px;
}

ul {
  list-style-type: none;
  display: flex;
  padding: 0;
}

li {
  margin: 0 10px;
}

a {
  color: white;
  text-decoration: none;
  transition: color 0.3s;
  font-size: 25px; /* Увеличение размера текста */
}

a:hover {
  color: #B01C1C;
}


  