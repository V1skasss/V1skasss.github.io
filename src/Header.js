import { Link } from 'react-router-dom';
import './Header.css';

const Header = () => {
    const links = [
        { text: 'Главная', url: '/' },
        { text: 'Концерты', url: '/Concerts' },
        { text: 'Услуги', url: '/Uslugi' },
        { text: 'О нас', url: '/About' },
    ];

    return (
        <header className="header">
            <div className="logo-and-title">
            <div className="logo">
                <img src="logo.webp" alt="Логотип" />
            </div>
            <span className="title">Три Дня Дождя</span>
            </div>
            <nav>
            <ul>
            {links.map((link, index) => (
                <li key={index}>
                <Link to={link.url}>{link.text}</Link>
                </li>
            ))}
            </ul>
        </nav>
        </header>
    );
};


export default Header;