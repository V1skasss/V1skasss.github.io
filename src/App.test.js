import renderer from 'react-test-renderer';
import Header from './Header';

test('Снапшот шапку', () => {
  let header = renderer.create(<Header />);
  let headerJson = header.toJSON();
  expect(headerJson).toMatchSnapshot();
});