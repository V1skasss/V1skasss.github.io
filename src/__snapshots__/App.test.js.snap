// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Снапшот шапку 1`] = `
<header>
  <nav
    className="flex justify-around"
  >
    <a
      className="text-pink hover:text-darkPink"
      href="#"
    >
      Link
    </a>
    <a
      className="text-pink hover:text-darkPink"
      href="#"
    >
      Link
    </a>
    <a
      className="text-pink hover:text-darkPink"
      href="#"
    >
      Link
    </a>
    <a
      className="text-pink hover:text-darkPink"
      href="#"
    >
      Link
    </a>
    <a
      className="text-pink hover:text-darkPink"
      href="#"
    >
      Link
    </a>
  </nav>
</header>
`;
